packages/config/src/Devices.ts

```diff
@@ -649,7 +649,7 @@ Parameter #${parameterNumber}: allowManualEntry must be a boolean!`,
 		if (
 			isArray(definition.options) &&
 			!definition.options.every(
-				(opt) =>
+				(opt: any) =>
 					isObject(opt) &&
 					typeof (opt as any).label === "string" &&
 					typeof (opt as any).value === "number",
```

Comment:
The `as any` here should now be unnecessary.

Reply:
I can remove it

Reply:
I wasn't able to remove it. it says:
```
Devices.ts:652:6 - error TS7006: Parameter 'opt' implicitly has an 'any' type.
652  			(opt) =>
     			 ~~~
```